lib/content-services/src/lib/document-list/models/file-auto-download-actions.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum FileAutoDownloadActionsEnum {
```

Comment:
missing JSDoc for public api

Reply:
Added licence JSDoc to file

Reply:
I don't see any comments in the PR, did you commit the changes?

Reply:
By JSDoc I mean comments for class and public class members, this is not present. You just have a license header there.